xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -1412,8 +1414,14 @@ bool CDecoder::OpenDecoder()
   m_avD3D11Context->video_context = nullptr;
   m_avD3D11Context->surface_count = m_refs;
 
+  DXGI_ADAPTER_DESC AIdentifier = {};
+  DX::DeviceResources::Get()->GetAdapterDesc(&AIdentifier);
+
+  // use true shared buffers on Intel
+  bool trueShared = m_dxvaContext->IsContextShared() && AIdentifier.VendorId == PCIV_Intel;
```

Comment:
No. Is used shared texture but is not shared same texture is used for decoder. Is shared only a "copy texture".   That has not changed and continues as it was "almost":

Before:
```
DECODER TEXTURE    =>    COPY TEXTURE
With shared flag         With shared flag
```

After:
```
DECODER TEXTURE    =>    COPY TEXTURE
No shared flag           With shared flag
```

Decoder texture not need shared flag in Nvidia because only is shared Copy texture